From 94ecf754996bfe941c733ee32bfad118e04b4c20 Mon Sep 17 00:00:00 2001
From: Mitali Vagadiya <vmnagajibhai@habana.ai>
Date: Mon, 21 Oct 2024 07:48:25 +0300
Subject: [PATCH] [SW-200837] dl: add packages in dl directory

As part of BR 2024.08 update, new packages are downloaded as part
of compilation. To improvide overall compilation time, this patch
adds these packages as part of dl folder.
---

diff --git a/dl/acl/acl-2.3.2.tar.xz b/dl/acl/acl-2.3.2.tar.xz
new file mode 100644
index 0000000..cba8d60
--- /dev/null
+++ b/dl/acl/acl-2.3.2.tar.xz
Binary files differ
diff --git a/dl/attr/attr-2.5.2.tar.xz b/dl/attr/attr-2.5.2.tar.xz
new file mode 100644
index 0000000..bad30c0
--- /dev/null
+++ b/dl/attr/attr-2.5.2.tar.xz
Binary files differ
diff --git a/dl/autoconf/autoconf-2.72.tar.xz b/dl/autoconf/autoconf-2.72.tar.xz
new file mode 100644
index 0000000..ac6d64a
--- /dev/null
+++ b/dl/autoconf/autoconf-2.72.tar.xz
Binary files differ
diff --git a/dl/bash/bash-5.2.21.tar.gz b/dl/bash/bash-5.2.21.tar.gz
new file mode 100644
index 0000000..4180121
--- /dev/null
+++ b/dl/bash/bash-5.2.21.tar.gz
Binary files differ
diff --git a/dl/busybox/busybox-1.36.1.tar.bz2 b/dl/busybox/busybox-1.36.1.tar.bz2
new file mode 100644
index 0000000..529defd
--- /dev/null
+++ b/dl/busybox/busybox-1.36.1.tar.bz2
Binary files differ
diff --git a/dl/cmake/cmake-3.30.0.tar.gz b/dl/cmake/cmake-3.30.0.tar.gz
new file mode 100644
index 0000000..4958022
--- /dev/null
+++ b/dl/cmake/cmake-3.30.0.tar.gz
Binary files differ
diff --git a/dl/dtc/dtc-1.7.0.tar.xz b/dl/dtc/dtc-1.7.0.tar.xz
new file mode 100644
index 0000000..c974b08
--- /dev/null
+++ b/dl/dtc/dtc-1.7.0.tar.xz
Binary files differ
diff --git a/dl/fakeroot/fakeroot_1.34.orig.tar.gz b/dl/fakeroot/fakeroot_1.34.orig.tar.gz
new file mode 100644
index 0000000..4e51bae
--- /dev/null
+++ b/dl/fakeroot/fakeroot_1.34.orig.tar.gz
Binary files differ
diff --git a/dl/gettext-tiny/gettext-0.22.4.tar.xz b/dl/gettext-tiny/gettext-0.22.4.tar.xz
new file mode 100644
index 0000000..a5582e6
--- /dev/null
+++ b/dl/gettext-tiny/gettext-0.22.4.tar.xz
Binary files differ
diff --git a/dl/kmod/kmod-31.tar.xz b/dl/kmod/kmod-31.tar.xz
new file mode 100644
index 0000000..bf6595c
--- /dev/null
+++ b/dl/kmod/kmod-31.tar.xz
Binary files differ
diff --git a/dl/libopenssl/openssl-3.3.2.tar.gz b/dl/libopenssl/openssl-3.3.2.tar.gz
new file mode 100644
index 0000000..eeccaf3
--- /dev/null
+++ b/dl/libopenssl/openssl-3.3.2.tar.gz
Binary files differ
diff --git a/dl/libzlib/zlib-1.3.1.tar.xz b/dl/libzlib/zlib-1.3.1.tar.xz
new file mode 100644
index 0000000..2083837
--- /dev/null
+++ b/dl/libzlib/zlib-1.3.1.tar.xz
Binary files differ
diff --git a/dl/lzop/lzop-1.04.tar.gz b/dl/lzop/lzop-1.04.tar.gz
new file mode 100644
index 0000000..b35e736
--- /dev/null
+++ b/dl/lzop/lzop-1.04.tar.gz
Binary files differ
diff --git a/dl/mtd/mtd-utils-2.2.0.tar.bz2 b/dl/mtd/mtd-utils-2.2.0.tar.bz2
new file mode 100644
index 0000000..5d45bbc
--- /dev/null
+++ b/dl/mtd/mtd-utils-2.2.0.tar.bz2
Binary files differ
diff --git a/dl/ncurses/ncurses-6.4-20230603.tar.gz b/dl/ncurses/ncurses-6.4-20230603.tar.gz
new file mode 100644
index 0000000..bf4f23f
--- /dev/null
+++ b/dl/ncurses/ncurses-6.4-20230603.tar.gz
Binary files differ
diff --git a/dl/pciutils/pciutils-3.13.0.tar.xz b/dl/pciutils/pciutils-3.13.0.tar.xz
new file mode 100644
index 0000000..5b6ae2f
--- /dev/null
+++ b/dl/pciutils/pciutils-3.13.0.tar.xz
Binary files differ
diff --git a/dl/readline/readline-8.2.tar.gz b/dl/readline/readline-8.2.tar.gz
new file mode 100644
index 0000000..ca5396a
--- /dev/null
+++ b/dl/readline/readline-8.2.tar.gz
Binary files differ
diff --git a/dl/tar/tar-1.35.cpio.gz b/dl/tar/tar-1.35.cpio.gz
new file mode 100644
index 0000000..eecf154
--- /dev/null
+++ b/dl/tar/tar-1.35.cpio.gz
Binary files differ
